### DISCUSSION TITLE
Add more cryptocurrency address

### DISCUSSION THREAD

**Discussion by BluePositive** [samuel-lucas6/Kryptor#39] [Ideas]
First I want to express my appreciation for the project, It is a very powerful easy-to-use tool with gr8 documentation.

I would like to donate some cryptocurrency other than Monero (which I can't buy in my region) I would suggest adding an Ethereum address, that will enable you to accept with 1 address donations on all EVM compatible rollups/chains (Arbitrum, Optimism, Polygon, Fantom, etc..)

Thanks a lot!
Keep ciphering!

**Comment by samuel-lucas6** [comment]
Thank you very much! I will look into it. I did have a Bitcoin address, but I never set up an easy way to withdraw funds if anything were to be donated.

**Reply by samuel-lucas6** [reply]
@BluePositive I've set up a wallet after doing some research. Here's the address: 0x1d140cdce6c83de556638acab209ea3f119da02d. I will add it to the website.

**Reply by BluePositive** [reply]
Gr8 you can now receive donations on all EVM compatible rollups/chains for all types of coins/tokens.
You can use [Zerion](https://zerion.io/) or [Zapper](https://zapper.fi/) to monitor donations on (almost) all EVM compatible rollups/chains.

(That is only if you have access to the private key for this address like it is the case with [MetaMask](https://metamask.io/), [Ledger]( https://www.ledger.com/) or any **non-custodial wallet** if you don't have the private key for this address like it is the case with [Coinbase](https://www.coinbase.com/), [Gemini](https://www.gemini.com/) or any **custodial wallet/exchange** you will only have access to donations for the specific coin/token (and only on the chain) that the exchange is specifying for this address.)
